odo/backends/sql.py

```diff
@@ -692,6 +692,7 @@ def select_or_selectable_to_frame(el, bind=None, dshape=None, **kwargs):
             parse_dates=datetime_fields,
             dtype=other_dtypes,
             skip_blank_lines=False,
+            escapechar=kwargs.get('escapechar', '\\'),
```

Comment:
This is used to handle quoting of the quotechar even when quoting is set to `QUOTE_MINIMAL`. We tell postgres to use this same quotechar so it escapes the quotes with a backslash (by default) when generating the csv. If this is removed test to and from dataframe string fail.